superset-frontend/packages/superset-ui-core/test/chart/fixtures/formData.ts

```diff
@@ -19,13 +19,14 @@
 
 /* eslint sort-keys: 'off' */
 /** The form data defined here is based on default visualizations packaged with Apache Superset */
+import { TimeGranularity } from '../../../src/time-format/types';
```

Comment:
@rusackas and @eric-briscoe and I were talking about uni-directional imports from packages to the rest of superset-frontend but not the other direction (to avoid circular imports). I'd say since this is just a fixture for a test that it may be simpler to just leave the string there, but looking for other opinions here too.

Reply:
I agree with @eschutho, we want to avoid core importing from anywhere else to avoid circular scenario.  We could just make it a string, or consider moving the time-format constants to core like we have with other types